packages/ui-markdown-editor/src/utilities/constants.js

```diff
@@ -37,61 +37,104 @@ export const DROPDOWN_STYLE = {
   width: '100px',
 };
 
-export const DROPDOWN_STYLE_H1 = {
+export const PARAGRAPH_STYLING = {
+  margin: '1rem 0rem 0rem 0rem'
+}
+
+export const H1_STYLING = {
   fontSize: '25px',
   lineHeight: '23px',
+  margin: '2rem 0rem 1rem 0rem',
   fontWeight: MISC_CONSTANTS.DROPDOWN_WEIGHT,
   color: MISC_CONSTANTS.DROPDOWN_COLOR,
   textAlign: 'center',
 };
 
-export const DROPDOWN_STYLE_H2 = {
+export const H2_STYLING = {
   fontSize: '25px',
   lineHeight: '23px',
+  margin: '1.8rem 0rem 0.9rem 0rem',
   fontWeight: MISC_CONSTANTS.DROPDOWN_WEIGHT,
   color: MISC_CONSTANTS.DROPDOWN_COLOR,
 };
 
-export const DROPDOWN_STYLE_H3 = {
+export const H3_STYLING = {
   fontSize: '20px',
   lineHeight: '20px',
+  margin: '1.6rem 0rem 0.8rem 0rem',
   fontWeight: MISC_CONSTANTS.DROPDOWN_WEIGHT,
   color: MISC_CONSTANTS.DROPDOWN_COLOR,
 };
 
-export const DROPDOWN_STYLE_H4 = {
+export const H4_STYLING = {
   fontSize: '18px',
   lineHeight: '18px',
+  margin: '1.4rem 0rem 0.7rem 0rem',
   fontWeight: MISC_CONSTANTS.DROPDOWN_WEIGHT,
   color: MISC_CONSTANTS.DROPDOWN_COLOR,
 };
 
-export const DROPDOWN_STYLE_H5 = {
-  fontSize: '16x',
+export const H5_STYLING = {
+  fontSize: '16px',
   lineHeight: '16px',
+  margin: '1.2rem 0rem 0.6rem 0rem',
   fontWeight: MISC_CONSTANTS.DROPDOWN_WEIGHT,
   color: MISC_CONSTANTS.DROPDOWN_COLOR,
 };
 
-export const DROPDOWN_STYLE_H6 = {
+export const H6_STYLING = {
   fontSize: '14px',
   lineHeight: '14px',
-  fontWeight: MISC_CONSTANTS.DROPDOWN_WEIGHT,
   color: '#949CA2',
   letterSpacing: '0.3px',
-  // This margin is to override default styling in chrome
-  margin: '0.7rem 0rem 1rem 0rem',
   textTransform: 'uppercase',
+  margin: '1.2rem 0rem 0.5rem 0rem',
+  fontWeight: MISC_CONSTANTS.DROPDOWN_WEIGHT,
+};
+
+export const TOOLBAR_DROPDOWN_STYLE_H1 = {
+  fontSize: '25px',
+  lineHeight: '23px',
+  fontWeight: MISC_CONSTANTS.DROPDOWN_WEIGHT,
+  color: MISC_CONSTANTS.DROPDOWN_COLOR,
+  textAlign: 'center',
+};
```

Comment:
to make sure that the editor headings and toolbar headings stay consistent, I think it would be better if the toolbar style objects spread the editor ones and only changed the properties that are different (I think the only difference is that the dropdown ones should have no margin, correct?). So this one for example would become...
```
export const TOOLBAR_DROPDOWN_STYLE_H1 = {
  ...H1_STYLING,
  margin: 0,
};
```